.github/workflows/build.yml

```diff
@@ -0,0 +1,22 @@
+name: Build
+
+on: pull_request
```

Comment:
Shall we do it on the master branch too?

```
on:
  pull_request:
  push:
    branches:
      - master
 ```

Reply:
Sure just added